src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-SYSBENCH-OLTP.json

```diff
@@ -0,0 +1,413 @@
+﻿{
+  "Description": "Sysbench Database Server Performance Workload",
```

Comment:
nit: Mention MySQL in it, since technically sysbench can measure other benchmarks.